### PR TITLE
Frdq 56

### PR DESCRIPTION
**Pull request by Krysik**
I just decrease the padding of outline button. I'm not completely sure that this is a good solution.

### CODE REVIEW

**Comment by hayuna** [comment]
<p>Total Coverage: <code>63.33</code></p>
<details><summary>Coverage report</summary>
<p>
<pre>
> frontend@0.1.0 test:coverage /home/runner/work/frontend/frontend
> npm test -- --coverage


> frontend@0.1.0 test /home/runner/work/frontend/frontend
> CI=true react-scripts test --env=jsdom "--coverage"

------------------------------------------|----------|----------|----------|----------|-------------------|
File                                      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
------------------------------------------|----------|----------|----------|----------|-------------------|
All files                                 |    63.33 |    85.71 |       50 |    63.33 |                   |
 src                                      |        0 |      100 |        0 |        0 |                   |
  App.tsx                                 |        0 |      100 |        0 |        0 |                 4 |
  index.tsx                               |        0 |      100 |      100 |        0 |                 7 |
  react-app-env.d.ts                      |        0 |        0 |        0 |        0 |                   |
 src/components/ExampleStorybookComponent |        0 |      100 |        0 |        0 |                   |
  ExampleStorybookComponent.model.ts      |        0 |        0 |        0 |        0 |                   |
  ExampleStorybookComponent.tsx           |        0 |      100 |        0 |        0 |              9,10 |
 src/components/atoms/Button              |    90.91 |    90.91 |      100 |    90.91 |                   |
  Button.model.ts                         |        0 |        0 |        0 |        0 |                   |
  Button.tsx                              |    90.91 |    90.91 |      100 |    90.91 |                51 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/Header              |      100 |      100 |      100 |      100 |                   |
  Header.tsx                              |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/List                |        0 |      100 |        0 |        0 |                   |
  List.model.ts                           |        0 |        0 |        0 |        0 |                   |
  List.tsx                                |        0 |      100 |        0 |        0 |          12,13,16 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/ListItem            |        0 |        0 |        0 |        0 |                   |
  ListItem.model.ts                       |        0 |        0 |        0 |        0 |                   |
  ListItem.tsx                            |        0 |        0 |        0 |        0 |             11,12 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SnackBar            |      100 |      100 |      100 |      100 |                   |
  SnackBar.tsx                            |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SocialButton        |      100 |      100 |      100 |      100 |                   |
  SocialButton.model.ts                   |        0 |        0 |        0 |        0 |                   |
  SocialButton.tsx                        |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/styles                               |       75 |      100 |      100 |       75 |                   |
  GlobalStyles.tsx                        |        0 |      100 |      100 |        0 |                 4 |
  colors.ts                               |      100 |      100 |      100 |      100 |                   |
  devices.ts                              |      100 |      100 |      100 |      100 |                   |
  typography.ts                           |      100 |      100 |      100 |      100 |                   |
------------------------------------------|----------|----------|----------|----------|-------------------|
</pre>
</p>
</details>

**Comment by hayuna** [comment]
<p>Total Coverage: <code>63.33</code></p>
<details><summary>Coverage report</summary>
<p>
<pre>
> frontend@0.1.0 test:coverage /home/runner/work/frontend/frontend
> npm test -- --coverage


> frontend@0.1.0 test /home/runner/work/frontend/frontend
> CI=true react-scripts test --env=jsdom "--coverage"

------------------------------------------|----------|----------|----------|----------|-------------------|
File                                      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
------------------------------------------|----------|----------|----------|----------|-------------------|
All files                                 |    63.33 |    85.71 |       50 |    63.33 |                   |
 src                                      |        0 |      100 |        0 |        0 |                   |
  App.tsx                                 |        0 |      100 |        0 |        0 |                 4 |
  index.tsx                               |        0 |      100 |      100 |        0 |                 7 |
  react-app-env.d.ts                      |        0 |        0 |        0 |        0 |                   |
 src/components/ExampleStorybookComponent |        0 |      100 |        0 |        0 |                   |
  ExampleStorybookComponent.model.ts      |        0 |        0 |        0 |        0 |                   |
  ExampleStorybookComponent.tsx           |        0 |      100 |        0 |        0 |              9,10 |
 src/components/atoms/Button              |    90.91 |    90.91 |      100 |    90.91 |                   |
  Button.model.ts                         |        0 |        0 |        0 |        0 |                   |
  Button.tsx                              |    90.91 |    90.91 |      100 |    90.91 |                51 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/Header              |      100 |      100 |      100 |      100 |                   |
  Header.tsx                              |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/List                |        0 |      100 |        0 |        0 |                   |
  List.model.ts                           |        0 |        0 |        0 |        0 |                   |
  List.tsx                                |        0 |      100 |        0 |        0 |          12,13,16 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/ListItem            |        0 |        0 |        0 |        0 |                   |
  ListItem.model.ts                       |        0 |        0 |        0 |        0 |                   |
  ListItem.tsx                            |        0 |        0 |        0 |        0 |             11,12 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SnackBar            |      100 |      100 |      100 |      100 |                   |
  SnackBar.tsx                            |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SocialButton        |      100 |      100 |      100 |      100 |                   |
  SocialButton.model.ts                   |        0 |        0 |        0 |        0 |                   |
  SocialButton.tsx                        |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/styles                               |       75 |      100 |      100 |       75 |                   |
  GlobalStyles.tsx                        |        0 |      100 |      100 |        0 |                 4 |
  colors.ts                               |      100 |      100 |      100 |      100 |                   |
  devices.ts                              |      100 |      100 |      100 |      100 |                   |
  typography.ts                           |      100 |      100 |      100 |      100 |                   |
------------------------------------------|----------|----------|----------|----------|-------------------|
</pre>
</p>
</details>

File: src/components/atoms/Button/Button.styled.ts [hunk 1]
```diff
@@ -46,24 +46,28 @@ const setIconPosition = (pos: IconPosition) => {
...
+    case 'transparent':
+      return colors.background.Primary40;
+    default:
+      return colors.background.Neutral40;
   }
-}
+};
 
-export const ButtonIcon = styled.div<{ type: ButtonType, iconPos: IconPosition }>`
+export const ButtonIcon = styled.div<{ type: ButtonType; iconPos: IconPosition }>`
```
Comment (hayuna) [inline src/components/atoms/Button/Button.styled.ts:64]: Are you sure it should be `;` instead of `,` ?

Comment (Marcin-tech-ocean) [reply src/components/atoms/Button/Button.styled.ts:64]: That was hint from eslint or prettier

Comment (hayuna) [reply src/components/atoms/Button/Button.styled.ts:64]: @hendrysiak is this correct?

**Comment by hayuna** [comment]
<p>Total Coverage: <code>53.33</code></p>
<details><summary>Coverage report</summary>
<p>
<pre>
> frontend@0.1.0 test:coverage /home/runner/work/frontend/frontend
> npm test -- --coverage


> frontend@0.1.0 test /home/runner/work/frontend/frontend
> CI=true react-scripts test --env=jsdom "--coverage"

------------------------------------------|----------|----------|----------|----------|-------------------|
File                                      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
------------------------------------------|----------|----------|----------|----------|-------------------|
All files                                 |    53.33 |    85.71 |       40 |    53.33 |                   |
 src                                      |        0 |      100 |        0 |        0 |                   |
  App.tsx                                 |        0 |      100 |        0 |        0 |                 4 |
  index.tsx                               |        0 |      100 |      100 |        0 |                 7 |
  react-app-env.d.ts                      |        0 |        0 |        0 |        0 |                   |
 src/components/ExampleStorybookComponent |        0 |      100 |        0 |        0 |                   |
  ExampleStorybookComponent.model.ts      |        0 |        0 |        0 |        0 |                   |
  ExampleStorybookComponent.tsx           |        0 |      100 |        0 |        0 |              9,10 |
 src/components/atoms/Button              |    90.91 |    90.91 |      100 |    90.91 |                   |
  Button.model.ts                         |        0 |        0 |        0 |        0 |                   |
  Button.tsx                              |    90.91 |    90.91 |      100 |    90.91 |                51 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/Header              |        0 |      100 |        0 |        0 |                   |
  Header.tsx                              |        0 |      100 |        0 |        0 |              9,10 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/List                |        0 |      100 |        0 |        0 |                   |
  List.model.ts                           |        0 |        0 |        0 |        0 |                   |
  List.tsx                                |        0 |      100 |        0 |        0 |          12,13,16 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/ListItem            |        0 |        0 |        0 |        0 |                   |
  ListItem.model.ts                       |        0 |        0 |        0 |        0 |                   |
  ListItem.tsx                            |        0 |        0 |        0 |        0 |             11,12 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SnackBar            |      100 |      100 |      100 |      100 |                   |
  SnackBar.tsx                            |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SocialButton        |      100 |      100 |      100 |      100 |                   |
  SocialButton.model.ts                   |        0 |        0 |        0 |        0 |                   |
  SocialButton.tsx                        |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/styles                               |       50 |      100 |      100 |       50 |                   |
  GlobalStyles.tsx                        |        0 |      100 |      100 |        0 |                 4 |
  colors.ts                               |      100 |      100 |      100 |      100 |                   |
  devices.ts                              |        0 |      100 |      100 |        0 |                 1 |
  typography.ts                           |      100 |      100 |      100 |      100 |                   |
------------------------------------------|----------|----------|----------|----------|-------------------|
</pre>
</p>
</details>

**Comment by hayuna** [comment]
<p>Total Coverage: <code>53.33</code></p>
<details><summary>Coverage report</summary>
<p>
<pre>
> frontend@0.1.0 test:coverage /home/runner/work/frontend/frontend
> npm test -- --coverage


> frontend@0.1.0 test /home/runner/work/frontend/frontend
> CI=true react-scripts test --env=jsdom "--coverage"

------------------------------------------|----------|----------|----------|----------|-------------------|
File                                      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
------------------------------------------|----------|----------|----------|----------|-------------------|
All files                                 |    53.33 |    85.71 |       40 |    53.33 |                   |
 src                                      |        0 |      100 |        0 |        0 |                   |
  App.tsx                                 |        0 |      100 |        0 |        0 |                 4 |
  index.tsx                               |        0 |      100 |      100 |        0 |                 7 |
  react-app-env.d.ts                      |        0 |        0 |        0 |        0 |                   |
 src/components/ExampleStorybookComponent |        0 |      100 |        0 |        0 |                   |
  ExampleStorybookComponent.model.ts      |        0 |        0 |        0 |        0 |                   |
  ExampleStorybookComponent.tsx           |        0 |      100 |        0 |        0 |              9,10 |
 src/components/atoms/Button              |    90.91 |    90.91 |      100 |    90.91 |                   |
  Button.model.ts                         |        0 |        0 |        0 |        0 |                   |
  Button.tsx                              |    90.91 |    90.91 |      100 |    90.91 |                51 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/Header              |        0 |      100 |        0 |        0 |                   |
  Header.tsx                              |        0 |      100 |        0 |        0 |              9,10 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/List                |        0 |      100 |        0 |        0 |                   |
  List.model.ts                           |        0 |        0 |        0 |        0 |                   |
  List.tsx                                |        0 |      100 |        0 |        0 |          12,13,16 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/ListItem            |        0 |        0 |        0 |        0 |                   |
  ListItem.model.ts                       |        0 |        0 |        0 |        0 |                   |
  ListItem.tsx                            |        0 |        0 |        0 |        0 |             11,12 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SnackBar            |      100 |      100 |      100 |      100 |                   |
  SnackBar.tsx                            |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SocialButton        |      100 |      100 |      100 |      100 |                   |
  SocialButton.model.ts                   |        0 |        0 |        0 |        0 |                   |
  SocialButton.tsx                        |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/styles                               |       50 |      100 |      100 |       50 |                   |
  GlobalStyles.tsx                        |        0 |      100 |      100 |        0 |                 4 |
  colors.ts                               |      100 |      100 |      100 |      100 |                   |
  devices.ts                              |        0 |      100 |      100 |        0 |                 1 |
  typography.ts                           |      100 |      100 |      100 |      100 |                   |
------------------------------------------|----------|----------|----------|----------|-------------------|
</pre>
</p>
</details>

**Comment by hayuna** [comment]
<p>Total Coverage: <code>48.48</code></p>
<details><summary>Coverage report</summary>
<p>
<pre>
> frontend@0.1.0 test:coverage /home/runner/work/frontend/frontend
> npm test -- --coverage


> frontend@0.1.0 test /home/runner/work/frontend/frontend
> CI=true react-scripts test --env=jsdom "--coverage"

------------------------------------------|----------|----------|----------|----------|-------------------|
File                                      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
------------------------------------------|----------|----------|----------|----------|-------------------|
All files                                 |    48.48 |    85.71 |    36.36 |    48.48 |                   |
 src                                      |        0 |      100 |        0 |        0 |                   |
  App.tsx                                 |        0 |      100 |        0 |        0 |                 6 |
  index.tsx                               |        0 |      100 |      100 |        0 |                 7 |
  react-app-env.d.ts                      |        0 |        0 |        0 |        0 |                   |
 src/assets                               |        0 |        0 |        0 |        0 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/ExampleStorybookComponent |        0 |      100 |        0 |        0 |                   |
  ExampleStorybookComponent.model.ts      |        0 |        0 |        0 |        0 |                   |
  ExampleStorybookComponent.tsx           |        0 |      100 |        0 |        0 |              9,10 |
 src/components/atoms/Button              |    90.91 |    90.91 |      100 |    90.91 |                   |
  Button.model.ts                         |        0 |        0 |        0 |        0 |                   |
  Button.tsx                              |    90.91 |    90.91 |      100 |    90.91 |                51 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/Header              |        0 |      100 |        0 |        0 |                   |
  Header.tsx                              |        0 |      100 |        0 |        0 |             10,11 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/List                |        0 |      100 |        0 |        0 |                   |
  List.model.ts                           |        0 |        0 |        0 |        0 |                   |
  List.tsx                                |        0 |      100 |        0 |        0 |          13,14,17 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/ListItem            |        0 |        0 |        0 |        0 |                   |
  ListItem.model.ts                       |        0 |        0 |        0 |        0 |                   |
  ListItem.tsx                            |        0 |        0 |        0 |        0 |             11,12 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SnackBar            |      100 |      100 |      100 |      100 |                   |
  SnackBar.tsx                            |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SocialButton        |      100 |      100 |      100 |      100 |                   |
  SocialButton.model.ts                   |        0 |        0 |        0 |        0 |                   |
  SocialButton.tsx                        |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/pages/Home                           |        0 |      100 |        0 |        0 |                   |
  Home.tsx                                |        0 |      100 |        0 |        0 |             19,20 |
  data.ts                                 |        0 |      100 |      100 |        0 |                 1 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/styles                               |       50 |      100 |      100 |       50 |                   |
  GlobalStyles.tsx                        |        0 |      100 |      100 |        0 |                 4 |
  colors.ts                               |      100 |      100 |      100 |      100 |                   |
  devices.ts                              |        0 |      100 |      100 |        0 |                 1 |
  typography.ts                           |      100 |      100 |      100 |      100 |                   |
------------------------------------------|----------|----------|----------|----------|-------------------|
</pre>
</p>
</details>

**Comment by hayuna** [comment]
<p>Total Coverage: <code>60.00</code></p>
<details><summary>Coverage report</summary>
<p>
<pre>
> frontend@0.1.0 test:coverage /home/runner/work/frontend/frontend
> npm test -- --coverage


> frontend@0.1.0 test /home/runner/work/frontend/frontend
> CI=true react-scripts test --env=jsdom "--coverage"

------------------------------------------|----------|----------|----------|----------|-------------------|
File                                      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
------------------------------------------|----------|----------|----------|----------|-------------------|
All files                                 |       60 |    95.24 |    58.33 |       60 |                   |
 src                                      |        0 |      100 |        0 |        0 |                   |
  App.tsx                                 |        0 |      100 |        0 |        0 |                 6 |
  index.tsx                               |        0 |      100 |      100 |        0 |                 7 |
  react-app-env.d.ts                      |        0 |        0 |        0 |        0 |                   |
 src/assets                               |        0 |        0 |        0 |        0 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/ExampleStorybookComponent |        0 |      100 |        0 |        0 |                   |
  ExampleStorybookComponent.model.ts      |        0 |        0 |        0 |        0 |                   |
  ExampleStorybookComponent.tsx           |        0 |      100 |        0 |        0 |              9,10 |
 src/components/atoms/Button              |    90.91 |    90.91 |      100 |    90.91 |                   |
  Button.model.ts                         |        0 |        0 |        0 |        0 |                   |
  Button.tsx                              |    90.91 |    90.91 |      100 |    90.91 |                57 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/Header              |        0 |      100 |        0 |        0 |                   |
  Header.tsx                              |        0 |      100 |        0 |        0 |             10,11 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/List                |      100 |      100 |      100 |      100 |                   |
  List.model.ts                           |        0 |        0 |        0 |        0 |                   |
  List.tsx                                |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/ListItem            |      100 |      100 |      100 |      100 |                   |
  ListItem.model.ts                       |        0 |        0 |        0 |        0 |                   |
  ListItem.tsx                            |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/Paragraph           |        0 |      100 |        0 |        0 |                   |
  Paragraph.tsx                           |        0 |      100 |        0 |        0 |             10,11 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SnackBar            |      100 |      100 |      100 |      100 |                   |
  SnackBar.tsx                            |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SocialButton        |      100 |      100 |      100 |      100 |                   |
  SocialButton.model.ts                   |        0 |        0 |        0 |        0 |                   |
  SocialButton.tsx                        |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/pages/Home                           |        0 |      100 |        0 |        0 |                   |
  Home.tsx                                |        0 |      100 |        0 |        0 |             19,20 |
  data.ts                                 |        0 |      100 |      100 |        0 |                 1 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/styles                               |       50 |      100 |      100 |       50 |                   |
  GlobalStyles.tsx                        |        0 |      100 |      100 |        0 |                 4 |
  colors.ts                               |      100 |      100 |      100 |      100 |                   |
  devices.ts                              |        0 |      100 |      100 |        0 |                 1 |
  typography.ts                           |      100 |      100 |      100 |      100 |                   |
------------------------------------------|----------|----------|----------|----------|-------------------|
</pre>
</p>
</details>

**Comment by hayuna** [comment]
<p>Total Coverage: <code>62.86</code></p>
<details><summary>Coverage report</summary>
<p>
<pre>
> frontend@0.1.0 test:coverage /home/runner/work/frontend/frontend
> npm test -- --coverage


> frontend@0.1.0 test /home/runner/work/frontend/frontend
> CI=true react-scripts test --env=jsdom "--coverage"

------------------------------------------|----------|----------|----------|----------|-------------------|
File                                      |  % Stmts | % Branch |  % Funcs |  % Lines | Uncovered Line #s |
------------------------------------------|----------|----------|----------|----------|-------------------|
All files                                 |    62.86 |    95.24 |    58.33 |    62.86 |                   |
 src                                      |        0 |      100 |        0 |        0 |                   |
  App.tsx                                 |        0 |      100 |        0 |        0 |                 6 |
  index.tsx                               |        0 |      100 |      100 |        0 |                 7 |
  react-app-env.d.ts                      |        0 |        0 |        0 |        0 |                   |
 src/assets                               |        0 |        0 |        0 |        0 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/ExampleStorybookComponent |        0 |      100 |        0 |        0 |                   |
  ExampleStorybookComponent.model.ts      |        0 |        0 |        0 |        0 |                   |
  ExampleStorybookComponent.tsx           |        0 |      100 |        0 |        0 |              9,10 |
 src/components/atoms/Button              |    90.91 |    90.91 |      100 |    90.91 |                   |
  Button.model.ts                         |        0 |        0 |        0 |        0 |                   |
  Button.tsx                              |    90.91 |    90.91 |      100 |    90.91 |                57 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/Header              |        0 |      100 |        0 |        0 |                   |
  Header.tsx                              |        0 |      100 |        0 |        0 |             10,11 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/List                |      100 |      100 |      100 |      100 |                   |
  List.model.ts                           |        0 |        0 |        0 |        0 |                   |
  List.tsx                                |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/ListItem            |      100 |      100 |      100 |      100 |                   |
  ListItem.model.ts                       |        0 |        0 |        0 |        0 |                   |
  ListItem.tsx                            |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/Paragraph           |        0 |      100 |        0 |        0 |                   |
  Paragraph.tsx                           |        0 |      100 |        0 |        0 |             10,11 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SnackBar            |      100 |      100 |      100 |      100 |                   |
  SnackBar.tsx                            |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/components/atoms/SocialButton        |      100 |      100 |      100 |      100 |                   |
  SocialButton.model.ts                   |        0 |        0 |        0 |        0 |                   |
  SocialButton.tsx                        |      100 |      100 |      100 |      100 |                   |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/pages/Home                           |        0 |      100 |        0 |        0 |                   |
  Home.tsx                                |        0 |      100 |        0 |        0 |             19,20 |
  data.ts                                 |        0 |      100 |      100 |        0 |                 1 |
  index.ts                                |        0 |        0 |        0 |        0 |                   |
 src/styles                               |       75 |      100 |      100 |       75 |                   |
  GlobalStyles.tsx                        |        0 |      100 |      100 |        0 |                 4 |
  colors.ts                               |      100 |      100 |      100 |      100 |                   |
  devices.ts                              |      100 |      100 |      100 |      100 |                   |
  typography.ts                           |      100 |      100 |      100 |      100 |                   |
------------------------------------------|----------|----------|----------|----------|-------------------|
</pre>
</p>
</details>

**Pull request merged**